src/modules/recipe/miscMultipleTemplates/miscMultipleTemplates.js

```diff
@@ -0,0 +1,15 @@
+import { LightningElement } from 'lwc';
+import templateOne from './templateOne.html';
+import templateTwo from './templateTwo.html';
+
+export default class MiscMultipleTemplates extends LightningElement {
+    templateOne = true;
```

Comment:
The property should be different than the imported template name. Like `showTemplateOne`.